packages/console/src/pages/Connectors/components/Guide/index.module.scss

```diff
@@ -36,7 +36,7 @@
     overflow-x: auto;
     justify-content: center;
     min-width: min-content;
-    padding-top: _.unit(2);
+    padding: _.unit(2) _.unit(18) 0;
```

Comment:
just double check confirmed with designer?

Reply:
double checked with the designer, should keep the bottom padding otherwise the scroll bar could reach the border.

Reply:
Well I remember they said that the scrollbar of the right panel should be sticking right next to the edge of the screen, so I think there are still work to do

Reply:
I was going to fix the scrollbar issue in the upcoming PR.

Reply:
i'm good